specification/communication/data-plane/PhoneNumbers/stable/2022-03-31/phonenumbers.json

```
@@ -0,0 +1,1440 @@
+            "default": 0
+          },
+          {
+            "in": "query",
+            "name": "top",
+            "description": "An optional parameter for how many entries to return, for pagination purposes. The default value is 100.",
+            "type": "integer",
+            "format": "int32",
+            "default": 100
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/PhoneNumberCountries"
+            }
+          },
+          "default": {
+            "description": "Failure",
+            "schema": {
+              "$ref": "../../../Common/stable/2021-03-07/common.json#/definitions/CommunicationErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get PhoneNumber Search Result": {
+            "$ref": "./examples/GetCountries.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink",
+          "itemName": "phoneNumberCountries"
+        }
+      }
+    },
+    "/availablePhoneNumbers/countries/{countryCode}/localities": {
+      "get": {
+        "tags": [
+          "Browse for available countries, cities and area codes"
+        ],
+        "summary": "Gets the list of cities or towns with available phone numbers.",
+        "operationId": "PhoneNumbers_ListAvailableCities",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "header",
+            "name": "accept-language",
+            "description": "The locale to display in the localized fields in the response.",
+            "type": "string"
+          },
+          {
+            "in": "path",
+            "name": "countryCode",
+            "description": "The ISO 3166-2 country code, e.g. US.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "skip",
+            "description": "An optional parameter for how many entries to skip, for pagination purposes. The default value is 0.",
+            "type": "integer",
+            "format": "int32",
+            "default": 0
+          },
+          {
+            "in": "query",
+            "name": "top",
+            "description": "An optional parameter for how many entries to return, for pagination purposes. The default value is 100.",
+            "type": "integer",
+            "format": "int32",
+            "default": 100
+          },
+          {
+            "in": "query",
+            "name": "administrativeDivision",
+            "description": "An optional parameter for the name of the state or province in which to search for the area code.",
+            "type": "string"
+          },
```

The trailing comma here is tripping up the pipeline checks.
```suggestion
          }
```